HealthcareSystem/PatientWebApp/Controllers/EventController.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using PatientWebApp.Auth;
+using PatientWebApp.Controllers.Adapter;
+using PatientWebApp.DTOs;
+using RestSharp;
+
+namespace PatientWebApp.Controllers
+{
+    [Authorize]
+    [Route("api/[controller]")]
+    [ApiController]
+    public class EventController : ControllerBase
+    {
+
+        [Authorize(Roles = UserRoles.Admin)]
+        [HttpGet]
+        public IActionResult GetAll()
+        {
+            return RequestAdapter.SendRequestWithoutBody("http://localhost:58828", "/api/patientSchedulingEvent", Method.GET);
```

Comment:
I added the event service to the configuration, you can getting from the ServiceSettings.

Reply:
Resolved.